### ISSUE TITLE
TypeError: __init__() missing 1 required positional argument: 'policies'

### ISSUE THREAD

**Issue by jayvdb** [d9pouces/djt_csp#1]
I'm packaging for openSUSE, and most tests are failing with this.

```py
[   41s] ============================= test session starts ==============================
[   41s] platform linux -- Python 3.6.12, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /usr/bin/python3.6
[   41s] cachedir: .pytest_cache
[   41s] rootdir: /home/abuild/rpmbuild/BUILD/djt_csp-0.9.7
[   42s] collecting ... collected 6 items
[   42s] 
[   42s] tests_djt_csp/test_parse_csp.py::TestCSPParser::test_cspparser_1 FAILED  [ 16%]
[   42s] tests_djt_csp/test_parse_csp.py::TestCSPParser::test_cspparser_2 FAILED  [ 33%]
[   42s] tests_djt_csp/test_parse_csp.py::TestCSPParser::test_cspparser_3 FAILED  [ 50%]
[   42s] tests_djt_csp/test_parse_csp.py::TestCSPParser::test_cspparser_4 FAILED  [ 66%]
[   42s] tests_djt_csp/test_parse_csp.py::TestCSPParser::test_cspparser_5 FAILED  [ 83%]
[   42s] tests_djt_csp/test_parse_csp.py::TestCSPParser::test_cspparser_6 PASSED  [100%]
[   42s] 
[   42s] =================================== FAILURES ===================================
[   42s] ________________________ TestCSPParser.test_cspparser_1 ________________________
[   42s] 
[   42s] self = <tests_djt_csp.test_parse_csp.TestCSPParser testMethod=test_cspparser_1>
[   42s] 
[   42s]     def test_cspparser_1(self):
[   42s] >       parser = CSPParser()
[   42s] E       TypeError: __init__() missing 1 required positional argument: 'policies'
...
```

**Comment by d9pouces** [comment]
Should now be fixed in 0.9.8

**Comment by jayvdb** [comment]
Thanks.  Confirmed.  Now awaiting PyPI release.
